horace_core/sqw/@rundatah/private/calc_projections_.m

```diff
@@ -29,13 +29,13 @@
 %   urange      [2 x 4] array containing the full extent of the data in crystal Cartesian
 %              coordinates and energy transfer; first row the minima, second row the
 %              maxima.
-%   pix         [9 x npix] array of pixel information:
-%                   pix(1:4,:)  coordinates in crystal Cartesian coordinates and energy
-%                   pix(5,:)    run index: alway unity from this routine
-%                   pix(6,:)    detecetor index
-%                   pix(7,:)    energy bin index
-%                   pix(8,:)    signal
-%                   pix(9,:)    error squared
+%   pix         PixelData object with attributes:
```

Comment:
since this argument is now an object; do you still need to document its content?